pylint/checkers/imports.py

```diff
@@ -767,7 +767,9 @@ def _get_imported_module(self, importnode, modname):
                 return None
             self.add_message("relative-beyond-top-level", node=importnode)
         except astroid.AstroidSyntaxError as exc:
-            message = f"Cannot import {modname!r} due to syntax error {str(exc.error)!r}"  # pylint: disable=no-member; false positive
+            message = (
+                f"Cannot import {modname!r} due to syntax error {str(exc.error)!r}"
```

Comment:
```suggestion
                f"Cannot import {modname} due to syntax error {exc.error}"
```
How about that ?

Reply:
I'm reluctant to merge this in without a test, i.e. we should probably do it separately. What do you think? (Clearly at one point exc.error may not have been a string or could have been arbitrarily set by a user.)

Reply:
Let's do this in a separate MR anyway, there's probably a ton of old %r that were transformed to fstring without refactoring.